homeassistant/components/epson/media_player.py

```diff
@@ -122,44 +125,81 @@ async def async_service_handler(service):
     hass.services.async_register(
         DOMAIN, SERVICE_SELECT_CMODE, async_service_handler, schema=epson_schema
     )
+    return True
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the Epson projector."""
+    hass.async_create_task(
+        hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": SOURCE_IMPORT}, data=config
+        )
+    )
 
 
 class EpsonProjector(MediaPlayerEntity):
     """Representation of Epson Projector Device."""
 
-    def __init__(self, websession, name, host, port, encryption):
+    def __init__(self, websession, name, host, port, timeout_scale, entry_id):
         """Initialize entity to control Epson projector."""
         self._name = name
-        self._projector = epson.Projector(host, websession=websession, port=port)
+        self._projector = epson.Projector(
```

Comment:
You're creating another object while you already had one created inside async_setup_entry ?